source-map.bs

```diff
@@ -202,22 +203,22 @@ following structure:
 ```
 
 * <dfn><code>version</code></dfn> is the version field which must always be the number
-    `3` as an integer. The source map may be rejected in case of a value different from `3`.
+    `3` as an integer. The source map may be rejected if the field has any other value.
 * <dfn><code>file</code></dfn> is an optional name of the generated code
-    that this source map is associated with. It's not specified if this can
+    that the source map is associated with. It is not specified if this can
     be a URL, relative path name, or just a base name. As such it has a mostly informal
     character.
```

Comment:
I didn't adjust it yet because I'm not sure what we want to say, but I feel like the "As such it has a mostly informal character" isn't adding much information here. Should we say something about how generators can choose an appropriate interpretation for their context?

Reply:
Yes I prefer your solution to the current one which made little sense to me.

Reply:
I added some text about generators replacing this sentence. It's in a separate commit in case it's too close to non-editorial and there are any objections to changing it now.